ui/src/app/views/profile-page/component.spec.ts

```diff
@@ -124,4 +124,49 @@ describe('ProfilePageComponent', () => {
       fixture.debugElement.queryAll(By.css('#currentPositionInvalid')).length).toBeTruthy();
   }));
 
+  it('handles empty givenName field', fakeAsync( () => {
+      simulateInput(
+          fixture, fixture.debugElement.query(By.css('#givenName')), randomString(0));
```

Comment:
nit: `s/randomString(0)/''/` ?

Reply:
Not addressed, if you disagree or do not understand please comment (else I must assume you just missed it)

Reply:
sorry i missed this!  not sure what you mean though?

Reply:
`randomString(0)` is the same thing as the empty string.

Reply:
Thanks Greg, Jackie and I just synced on Slack as well on this and the boxed type thread.

For other observers, my above syntax was a sed replacement, i.e. suggesting substitution of `randomString(0)` with `''` throughout.